src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorDataSourceRetriever.java

```diff
@@ -0,0 +1,221 @@
+package org.broadinstitute.hellbender.tools.funcotator;
+
+import org.apache.commons.codec.digest.DigestUtils;
+import org.apache.commons.io.FileUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.BetaFeature;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.nio.NioFileCopier;
+import picard.cmdline.programgroups.VariantEvaluationProgramGroup;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Path;
+
+/**
+ * Tool to download the latest data sources for {@link Funcotator}.
+ * Created by jonn on 8/23/18.
+ */
+@CommandLineProgramProperties(
+        summary = "Download the Broad Institute pre-packaged data sources for the somatic or germline use case for Funcotator.",
+        oneLineSummary = "Data source downloader for Funcotator.",
+        programGroup = VariantEvaluationProgramGroup.class
+)
+@DocumentedFeature
+@BetaFeature
+public class FuncotatorDataSourceRetriever extends CommandLineProgram {
```

Comment:
I think `FuncotatorDataSourceDownloader` might be a slightly better name.

Reply:
Fixed.